src/video_core/renderer_vulkan/vk_shader_hle.cpp

```diff
@@ -5,16 +5,16 @@
 #include "video_core/renderer_vulkan/vk_scheduler.h"
 #include "video_core/renderer_vulkan/vk_shader_hle.h"
 
-#include "vk_rasterizer.h"
+extern std::unique_ptr<AmdGpu::Liverpool> liverpool;
 
 namespace Vulkan {
 
 static constexpr u64 COPY_SHADER_HASH = 0xfefebf9f;
 
-bool ExecuteCopyShaderHLE(const Shader::Info& info, const AmdGpu::Liverpool::Regs& regs,
-                          Rasterizer& rasterizer) {
+bool ExecuteCopyShaderHLE(const Shader::Info& info, Rasterizer& rasterizer) {
```

Comment:
You can pass a reference to ComputeProgram here instead of relying on extern liverpool